Assets/GothicVR/Scripts/Creator/MeshCreator.cs

```diff
@@ -139,12 +139,18 @@ public GameObject Create(string objectName, PxMultiResolutionMeshData mrm, Vecto
             rootGo.SetParent(parent);
             SetPosAndRot(rootGo, position, rotation);
 
+            if (mrm == null)
+            {
+                Debug.LogError("No mesh data was found for: " + objectName);
+                return rootGo;
```

Comment:
If we have no mesh I would recommend we return null instead of the empty GameObject. This way we can easily handle in the calling function what to do.

PS: If you want to return null, please take care, that you did Destroy(rootGo) otherwise we have garbage in the scene.

Reply:
Moved the return to before the GO is created so we don't have to clean it up :D